include/server.h

```diff
@@ -65,14 +66,14 @@ namespace kiwix
         { m_blockExternalLinks = blockExternalLinks; }
        void setIpMode(IpMode mode) { m_ipMode = mode; }
        int getPort();
-       std::string getAddress();
+       std::vector<std::string> getAddresses();
```

Comment:
Make `const` (and also please fix `const`-correctness of other getters in a separate commit)